tests/Integration/Behaviour/Features/Scenario/Product/SpecificPrice/edit_specific_price.feature

```diff
@@ -0,0 +1,191 @@
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+      | shop group            | 42     |
+      | shop                  | 51     |
+      | currency              | 69     |
+      | country               | 21     |
+      | group                 | 33     |
+      | customer              | 99     |
+    When I edit specific price "price1" with following details:
+      | shop group |  |
+      | shop       |  |
+      | currency   |  |
+      | country    |  |
+      | group      |  |
+      | customer   |  |
+    Then specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+      | shop group            |        |
+      | shop                  |        |
+      | currency              |        |
+      | country               |        |
+      | group                 |        |
+      | customer              |        |
+
+  Scenario: I edit specific price relations
+    Given specific price price1 should have following details:
+      | specific price detail | value  |
+      | reduction type        | amount |
+      | reduction value       | 12.56  |
+      | includes tax          | true   |
+      | price                 | 45.78  |
+      | from quantity         | 1      |
+      | from                  |        |
+      | to                    |        |
+    When I edit specific price "price1" with following details:
+      | from | 2021-01-01 10:00:00 |
+      | to   | 2021-01-01 11:00:00 |
+    Then specific price price1 should have following details:
+      | specific price detail | value               |
+      | reduction type        | amount              |
+      | reduction value       | 12.56               |
+      | includes tax          | true                |
+      | price                 | 45.78               |
+      | from quantity         | 1                   |
+      | from                  | 2021-01-01 10:00:00 |
+      | to                    | 2021-01-01 11:00:00 |
+    When I edit specific price "price1" with following details:
+      | from | 0000-00-00 00:00:00 |
+      | to   | 0000-00-00 00:00:00 |
+    Then specific price price1 should have following details:
+      | specific price detail | value               |
+      | reduction type        | amount              |
+      | reduction value       | 12.56               |
+      | includes tax          | true                |
+      | price                 | 45.78               |
+      | from quantity         | 1                   |
+      | from                  | 0000-00-00 00:00:00 |
```

Comment:
```suggestion
      | from                  |                          |
```

This value should also be tested, we need to validate that the specific price has turned back to the same state it had in the initial test (L159 first `Given` state) Both checks are important I think, the explicit 0000 thing and the "empty" value